crates/re_log_types/src/component_types/instance_key.rs

```diff
@@ -63,6 +63,16 @@ impl InstanceKey {
     pub fn specific_index(self) -> Option<InstanceKey> {
         self.is_specific().then_some(self)
     }
+
+    /// Creates a new [`InstanceKey`] that identifies a 2d coordinate.
+    pub fn from_2d_image_coordinate(x: u32, y: u32, image_width: u64) -> Self {
```

Comment:
you can make it more symmetrical with `to_2d_image_coordinate`:

```suggestion
    pub fn from_2d_image_coordinate([x, y]: [u32; 2], image_width: u64) -> Self {
```